www/strf03/cv08/users.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+require 'db.php';
+
+require 'admin_required.php';
+
+$stmt = $db->prepare("SELECT * FROM users");
+
+$stmt ->execute();
+$users = $stmt->fetchAll();
+
+?>
+<?php require __DIR__ . '/incl/header.php' ?>
+<table class="table table-hover">
+    <thead class="thead-dark">
+    <tr>
+        <th scope="col">#</th>
+        <th scope="col">Email</th>
+        <th scope="col">Privilege</th>
+    </tr>
+    </thead>
+    <tbody>
+<?php
+    foreach ($users as $user) {
+        echo '<tr>';
```

Comment:
html nevypisovat pomocí echo

Reply:
jo, tak to mám i teď v tom world clock. Ještě to pak upravím. Z jakého důvodu by se to nemělo dělat?

Reply:
to je z důvodu zachování přehlednosti a konzistence, je dobrý se zvyknout a udělat pořádek jak to jde